packages/esm-patient-search-app/src/patient-search-bar/patient-search-bar.component.tsx

```diff
@@ -27,29 +29,41 @@ const PatientSearchBar = React.forwardRef<HTMLInputElement, React.PropsWithChild
     );
 
     const handleSubmit = useCallback(
-      (event: React.FormEvent<HTMLFormElement>) => {
+      (event) => {
         event.preventDefault();
         if (searchTerm && searchTerm.trim()) {
           onSubmit(searchTerm.trim());
+        } else {
+          setIsInputClicked(true);
+          inputRef.current.focus();
         }
       },
       [onSubmit, searchTerm],
     );
 
+    useEffect(() => {
+      if (isInputClicked) {
+        const timeout = setTimeout(() => {
+          setIsInputClicked(false);
+        }, 5000);
+        return () => clearTimeout(timeout);
+      }
+    }, [isInputClicked]);
+
     return (
       <form onSubmit={handleSubmit} className={styles.searchArea}>
         {/* data-tutorial-target attribute is essential for joyride in onboarding app ! */}
         <Search
           autoFocus
-          className={styles.patientSearchInput}
+          className={`${styles.patientSearchInput} ${isInputClicked ? styles.darkPlaceholder : ''}`}
```

Comment:
We generally use the `classnames` library for stuff like this:

```suggestion
          className={classNames('styles.patientSearchInput', { [styles.darkPlaceholder]: isInputClicked })}
```

Reply:
Thanks !